docs/model-types.mdx

```diff
@@ -0,0 +1,467 @@
+---
+title: ML Models in MindsDB
+---
+
+In this section, we go through the ML model types available in MindsDB. These are regression models, classification models, and time series models.
+
+## What is an ML Model?
+
+A machine learning (ML) model is a program trained using the available data in order to learn how to recognize patterns and behaviors and make decisions that lead to predicting future data.
```

Comment:
I would rephrase the start as:

> A supervised machine learning (ML) [...]

Reply:
Changed:

> ## What is an ML Model?
> 
> A machine learning (ML) model is a program trained using the available data in order to learn how to recognize patterns and behaviors to predict future data. There are various types of ML models that use different learning paradigms, but MindsDB models are all *supervised* because they learn from pairs of input data and expected output.
> 
> You input data into an ML model. It processes this input data, searching for patterns and correlations. After that, the ML model returns output data defined based on the input data.